3rdparty/glslang/glslang/HLSL/hlslParseHelper.cpp

```diff
@@ -9524,6 +9524,7 @@ bool HlslParseContext::isInputBuiltIn(const TQualifier& qualifier) const
     case EbvSampleMask:
     case EbvSamplePosition:
     case EbvViewportIndex:
+    case EbvBaryCoordEXT:
```

Comment:
You should  submit this patch to glslang repo. It's OK for now to be like this for bgfx needs.

Reply:
Will do.